### ISSUE TITLE
Adaptation to the web interface of Webots

### ISSUE THREAD

**Issue by omichel** [aiwc/test_world#19]
I have been studying how to adapt the AIWC simulation to the Webots Web Interface (WWI) used in https://robotbenchmark.net and it seems some modifications to the simulation are necessary:

Instead of creating the robots using the `Supervisor::importMFNodeFromString` method which is not yet supported by the WWI, we should rather have the robots already created (possibly hidden under the stadium seats) and have the supervisor move them on the field and change their attributes as needed before starting the game.

If you agree with this change, we will implement it for you in a new pull request.

**Comment by omichel** [comment]
We have been working on improving the Webots Web Interface so that it shouldn't be needed to change your supervisor controller finally. We are currently testing and will confirm it.

**Comment by chshong** [comment]
Great! Thank you.

**Comment by omichel** [comment]
I am closing this issue for now.
